src/components/street-geo.js

```diff
@@ -76,13 +92,46 @@ AFRAME.registerComponent('street-geo', {
       }
     }
 
-    // if state is not clipping, then disable it
     if (this.google3d) {
+      // Handle clipping updates
       if (data.enableClipping) {
         this.google3d.setAttribute('obb-clipping', '');
       } else {
         this.google3d.removeAttribute('obb-clipping');
       }
+
+      // Handle blending updates
+      if (data.blendingEnabled) {
+        if (data.blendMode) {
+          this.google3d.setAttribute(
+            'blending-opacity',
+            this.returnBlendMode(data.blendMode)
+          );
```

Comment:
I think you can put here
```js
      if (oldData.blendingEnabled === false) {
        const currentEl = this.google3d;
        this.el.removeChild(currentEl);
        this.google3d = null;
        this.google3dCreate();
      }
```
for the case where you changed `blendingEnabled` from false to true instead of doing your check `changedProps.length === 1 && changedProps.includes('blendingEnabled')` below.